src/reducers/entities.js

```diff
@@ -21,6 +21,23 @@ export const clearResourceTemplates = (state) => {
   return newState
 }
 
+// Takes current user and optional uri and removes resourceURI
+export const copyResourceToEditor = (state, action) => {
+  const newState = { ...state }
+  const uri = action.uri
+
+  if (uri !== undefined) {
```

Comment:
Copy from search is going to have to be handled a different way, as loading the resource should not be done in reducer. I suspect it can simply be done as an additional parameter on retrieveResource thunk that would omit setting the URI.

Reply:
👍